apps/participants/serializers.py

```diff
@@ -43,6 +43,6 @@ def validate_email(self, value):
 
     def save(self):
         email = self.validated_data.get('email')
-        return Participant.objects.create(user=User.objects.get(email=email),
-                                          status=Participant.ACCEPTED,
-                                          team=self.participant_team)
+        return Participant.objects.get_or_create(user=User.objects.get(email=email),
+                                                 status=Participant.ACCEPTED,
+                                                 team=self.participant_team)
```

Comment:
@gauthamzz can you please remove the extra spaces that you have added in line 47 and 48?